plugin/web/extensions/sidebar/modals/datacatalog/components/hooks.ts

```diff
@@ -177,6 +185,10 @@ export default () => {
     postMsg({ action: "initDataCatalog" }); // Needed to trigger sending selected dataset ids from Sidebar
   }, []);
 
+  useEffect(() => {
+    postMsg({ action: "getTreeFilterData" });
+  }, []);
```

Comment:
Why are you splitting this from initDataCatalog? Can't the data you are fetching with this action be considered "initialization" data?

Reply:
I do this because I don't want to get the desktop data for mobile components.

Reply:
https://github.com/eukarya-inc/reearth-plateauview/blob/3acd6f46a98b6101227bf307a54e701151507387/plugin/web/extensions/sidebar/popups/mobileDropdown/Catalog/index.tsx#L138

Reply:
Hmmm, but in the getTreeFilterData action you are checking if `isMobile`, and then doing either reearth.ui.postMessage or reearth.popup.postMessage. Since you are already doing that check, can't you just do that in the `initDataCatalog` action? 
Make sense or you still think having the two actions is better?

Reply:
I don't know, I think that having the two actions is better.

Reply:
Done!